specs/runtime-configuration/README.md

```diff
@@ -0,0 +1,100 @@
+
+I'm actually more convinced now that we shouldn't change it just because the existing terminology ("theme provider") and implementation (a component wrapping a chunk of the tree to override config) seems so common among design systems. I'll let reviewers give feedback.
+
+### Add a global configuration element
+
+Matches the API of the local configuration element but has the behavior of overriding the global default behavior. Can likely [share a base class as done in prototyping](https://github.com/ni/nimble/blob/theme-auto-2-revenge-of-theme-auto/packages/nimble-components/src/theme-base/index.ts#L10).
+
+This makes the global configuration defined by the global configuration element instead of reactive. Reactive global behavior can be restored by clearing the element configuration, for example `el.theme='light';` to override and `el.theme=undefined;` to unset and restore reactive behavior. 
+
+During prototyping in various prime days one thing I tried to address but found challenging was what the behavior should be if multiple global configuration elements were used on a page. I think it's unlikely to be an issue / could be something we try to address if it seems like something multiple clients are accidentally running into. The behavior will be whichever global configuration element most recently updated a configuration token will have set the value. Won't try and track them or guess which should be allowed or blocked, etc.
+
+Proposed name: `nimble-global-configuration-provider` (but should align as the `-global-` version of whatever the decided local configuration name is)
+
+Alternative:
+- If multiple global configuration providers are constructed log a warning to the console. This ended up just being annoying during testing where multiple are created and seems doubtful to offer real world value.
+
+### Enable reactive global configuration
+
+Have the `theme` configuration token react to the browser [`prefers-color-scheme`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme).
+
+Have the `direction` configuration token react via MutationObserver to the `<html>` `dir` attribute configuration. See [FAST discussion](https://github.com/microsoft/fast/issues/5547#issuecomment-1027419532) about how the direction token should align with the `dir` attribute. See [W3C dir recommendations](https://www.w3.org/International/questions/qa-html-dir#quickanswer).
+
+Note: This only addresses the top-level `<html dir>` definition and does not try to address the `dir` attribute placement on arbitrary children in the tree. Being responsive to the `<html dir>` seems like a strict improvement on current behavior and more sophisticated behavior can be investigated in the future.
```

Comment:
> Do you know if your proposed change would fix any of this or make any of it worse?

These changes only impact things which respond to the `direction` configuration token which is just the `DirectionalStyleSheetBehavior` which is just used by [tree-items](https://github.com/ni/nimble/blob/6289ce14c7a920fa28a6adb70ed419c68c229a07/packages/nimble-components/src/tree-item/styles.ts#L182). And looking briefly at that style I wonder if it could be reconfigured to better leverage flex / grid layout and avoid the left / right margin sizing and remove usage of the `DirectionalStyleSheetBehavior` anyway.

> Or would our stance still be that our rtl support is untested?

Yep, still untested and unsupported. Just some infrastructure that might be useful but maybe ultimately wouldn't actually be needed if we can express everything completely in flex / grid.